src/features/instrument/Instrument.tsx

```diff
@@ -45,7 +38,52 @@ export const Instrument = () => {
     part.loopEnd = "4m";
   }
 
-  // @TODO: maybe move to separate components, maybe same file
+  // @TODO: check arent a beter way to write each thing now (always create new player?), also check updates in tone
+  function playSample(sampleName: string) {
```

Comment:
I left `playPart` and `playSample` in parent component because using both in several components.
1) is there a better way how I could share reusable functions in general? 

2) would useCallback be a better option? In current case I would not have any dependencies on which one to memoize the function, so no benefit, right?

3) now I see that this function doesn't depend on anything from parent components, so I could move it outside Instrument after I define `mediumVolume`. With this change I would save 1 prop, are there more benefits doing that (e.g. what impact function declaration has during rendering)?

4) In general, is it ok I create here a function which is executed only in a child's scope? Any standards how such functions should be highlighted/separated from others?